torchmetrics/audio/sdr.py

```diff
@@ -0,0 +1,108 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Optional, Tuple
+
+from torch import Tensor, tensor
+
+from torchmetrics.functional.audio.sdr import sdr
+from torchmetrics.metric import Metric
+
+
+class SDR(Metric):
+    r"""SDR evaluates the average Signal to Distortion Ratio (SDR) [1] metric of preds and target. A wrapper for
+    the mir_eval.separation.bss_eval_sources function.
```

Comment:
Please convert into a note like here:
https://github.com/PyTorchLightning/metrics/blob/fef83a028880f2ad3e0c265b3e5bb8a184805798/torchmetrics/audio/pesq.py#L29-L30
```suggestion
    r"""SDR evaluates the average Signal to Distortion Ratio (SDR) [1] metric of preds and target.
```